Promise/Promise+Extras.swift

```diff
@@ -14,8 +14,15 @@ import Foundation
 struct PromiseCheckError: Error { }
 
 public enum Promises {
+    
     /// Wait for all the promises you give it to fulfill, and once they have, fulfill itself
     /// with the array of all fulfilled values.
+    /// - Parameter promises: Promises to wait for
+    /// - Returns: A Promise that is completed when all of the input promises finish.
+    ///
+    /// Note that `race()` is different to `all()` because `race()` will
+    /// blindly take the first promise to complete and returns that promise,
+    /// where `all()` will wait for every promise to complete and return all of them.
```

Comment:
```suggestion
    /// Note that `race()` is different to `all()` because `race()` will
    /// take the first promise to complete and returns that promise's results,
    /// where `all()` will wait for every promise to complete and return all of the.
    /// results or the first failure. Fails if any of the input promises fail.
```

Reply:
CML: `the.` → `the`

Reply:
Fixed in c9bc94b448931d667b510320c0b65e9096386dd2